assets/javascripts/kitten/components/tables/list-table/components/row.js

```diff
@@ -47,10 +47,10 @@ const StyledItemList = styled.ul`
   }
 `
 
-export const ListTableRow = ({ isHighlighted, children, ...others }) => {
+export const ListTableRow = ({ children, listProps, ...others }) => {
   return (
-    <StyledRow isHighlighted={isHighlighted}>
-      <StyledItemList {...others}>
+    <StyledRow {...others}>
+      <StyledItemList {...listProps}>
```

Comment:
En modifiant de place le `{ ...others }`, j'ai l'impression que ca provoque un breaking change du composant. Il faudrait faire l'inverse avec une prop `rowProps`.

Reply:
Je me suis dit que :
- le fix est suffisamment proche de la release pour que les éléments concernés ne soient pas encore problématiques
- logiquement, je m'attendrais que les props passées directement à `<ListTable.Row>` s'appliquent à l'élément principal, puis que les props des éléments internes soient transmis via des props spécifiques
- on garde la même logique qu'avec les props pour le `header`

Reply:
Vu que ca fait que 4 jours, en effet on peut se permettre de faire un fixe. Que penses-tu d'ajouter une ligne de fixe pour cette correction et le reste en feature pour qu'on comprenne bien ?

```
- Feature: Add `listProps` to `ListTable.Row`.
- Fix: Move `ListTable.Row` `others` propagation on `row` element instead of `list` element.
```

(avec un meilleur anglais :)